lib/ansible/modules/cloud/huawei/hwc_network_vpc.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 Huawei
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file at
+#     https://www.github.com/huaweicloud/magic-modules
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+###############################################################################
+# Documentation
+###############################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: hwc_network_vpc
+description:
+    - Represents an vpc resource.
+short_description: Creates a Huawei Cloud VPC
+version_added: 2.7
```

Comment:
I'm not sure why validate-modules didn't catch this, but 2.8 is the current version under development.

```suggestion
version_added: 2.8
```

If this isn't merged before 2.8 is frozen then it will need to be updated again to 2.9.

Reply:
Done in new patch https://github.com/ansible/ansible/pull/54102